recipes/pycdntroids/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pycentroids" %}
+{% set version = "0.2.0rc2" %}
+{% set sha256 = "642109bffe3c2733c06e897b91bc999ea0e960981fea8ba74ddce25e90faf5fe" %}
```

Comment:
This hash can be moved below to the `source` block as it's done in other recipes.